test/helpers/TaskQueue.spec.js

```diff
@@ -0,0 +1,145 @@
+const assert = require('assert');
+
+// Import class to test
+const TaskQueue = require('../../helpers/TaskQueue');
+
+describe('helpers > TaskQueue', async function () {
+  it('Resolves add method once task is complete', async function () {
+    const queue = new TaskQueue(1);
+
+    let foo;
+    const task = async () => {
+      // Timeout for 10 milliseconds
+      await new Promise((r) => { setTimeout(r, 10); });
+      foo = 'bar';
+      return 'baz';
+    };
+
+    const result = await queue.add({ task });
+    assert.equal(foo, 'bar', 'add method resolved before task completed');
+    assert.equal(result, 'baz', 'add method did not return result of task');
```

Comment:
Nitpick: 
```
assert.equal(
  foo,
  'bar',
  'add method did not return result of task'
);
```

There are a few more assert.equals on this page that need to follow this format

Reply:
Is that the preferred format even when the call is <80 characters total?

Reply:
When it's less than 80 chars, it's up to you.